docs/man-pages/condor_token_request.rst

```diff
@@ -0,0 +1,136 @@
+
+
+*condor_token_request*
+======================
+
+request a token from a remote daemon
```

Comment:
```suggestion
request a token from a remote daemon for the TOKEN authentication method
```

Reply:
or maybe "interactively request a token from a remote daemon for the TOKEN authentication method"?